src/trace-forwarder/Makefile

```diff
@@ -3,10 +3,47 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+##VAR BUILD_TYPE=release|debug type of rust build
+BUILD_TYPE = release
+
+##VAR ARCH=arch target to build (format: uname -m)
+ARCH = $(shell uname -m)
+##VAR LIBC=musl|gnu
+LIBC ?= musl
+ifneq ($(LIBC),musl)
+    ifeq ($(LIBC),gnu)
+        override LIBC = gnu
+    else
+        $(error "ERROR: A non supported LIBC value was passed. Supported values are musl and gnu")
+    endif
+endif
+
+ifeq ($(ARCH), ppc64le)
+    override ARCH = powerpc64le
+    override LIBC = gnu
+    $(warning "WARNING: powerpc64le-unknown-linux-musl target is unavailable")
+endif
+
+ifeq ($(ARCH), s390x)
+    override LIBC = gnu
+    $(warning "WARNING: s390x-unknown-linux-musl target is unavailable")
+endif
```

Comment:
We've got the same code here:

- https://github.com/kata-containers/kata-containers/blob/main/src/agent/Makefile#L33..L54

It might be better to move this logic into https://github.com/kata-containers/kata-containers/blob/main/utils.mk and then include that file here (and in the agent) like we do already for the top-level `Makefile` (https://github.com/kata-containers/kata-containers/blob/main/Makefile#L20).

Reply:
done